### PR TITLE
Change Taxes app into Avatax

### PR DESCRIPTION
**Pull request by lkostrowski**
Some time ago, Taxes app was divied into Avatax and Taxjar

Avatax is actively maintained by Saleor, Taxjar remains open-source and can be forked

This PR
1. Changes Taxes app docs into Avatax docs, renaming things and removing Taxjar context
2. Links Taxjar to source code
3. Add redirects

https://linear.app/saleor/issue/SHOPX-243/migrate-taxes-app-docs-into-taxjar-and-avatax

### CODE REVIEW

File: docs/developer/app-store/apps/avatax/configuration.mdx [hunk 1]
```diff
@@ -40,7 +34,7 @@ Fields that affect the created transactions:
 
 #### Document recording
 
-Taxes App creates AvaTax transactions on two events:
+AvaTax App creates AvaTax transactions on two events:
```
Comment (krzysztofzuraw) [inline docs/developer/app-store/apps/avatax/configuration.mdx:37]: **Question:** should it be `AvaTax` or `Avatax`?

Comment (lkostrowski) [reply docs/developer/app-store/apps/avatax/configuration.mdx:37]: I will change to AvaTax everywhere

File: docs/developer/app-store/apps/avatax/overview.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+_AvaTax_ is a Saleor app that allows delegating tax calculations to [Avatax](https://www.avalara.com/us/en/products/calculations.html). It can replace the default ([flat rates](developer/taxes.mdx#flat-rates)) tax calculation method in Saleor. It affects both the checkout and the order creation process.
+
+Avalara AvaTax is a cloud-based solution automating transaction tax calculations and the tax filing process. Avalara
+provides real-time tax calculation using tax content from more than 12,000 US taxing jurisdictions and over 200
+countries, insuring your transaction tax is calculated based on the most current tax rules.
+
+## Features
+
+Taxes App offers:
```
Comment (peelar) [inline docs/developer/app-store/apps/avatax/overview.mdx:21]:
```suggestion
AvaTax App offers:
```

File: docs/developer/app-store/apps/avatax/architecture.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+title: Architecture
+---
+
+## Webhook events
+
+The Taxes App uses the following webhook events for tax reporting and calculations:
```
Comment (peelar) [inline docs/developer/app-store/apps/avatax/architecture.mdx:7]:
```suggestion
The AvaTax App uses the following webhook events for tax reporting and calculations:
```

File: docs/developer/app-store/apps/avatax/overview.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+If you want to explore how the AvaTax App works, you can read the [Avatax App architecture](./architecture) article.
+
+## Configuration
+
+### Channels
+
+For a channel to appear on the _Available channels_ list, you must first select **"use tax app"** as its tax calculation method. You can do that on the _Tax Configuration_ page (_Configuration_ → _Taxes_).
+
+In the _Taxes_ page, select the channel and change the value of _Select the method of tax calculation_ to "_Use tax app_". Then save it and come back to the Taxes App.
```
Comment (peelar) [inline docs/developer/app-store/apps/avatax/overview.mdx:44]:
```suggestion
In the _Taxes_ page, select the channel and change the value of _Select the method of tax calculation_ to "_Use tax app_". Then save it and come back to the AvaTax App.
```

File: versioned_docs/version-3.x/developer/app-store/apps/avatax/architecture.mdx [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
+title: Architecture
+---
+
+## Webhook events
+
+The Taxes App uses the following webhook events for tax reporting and calculations:
```
Comment (peelar) [inline versioned_docs/version-3.x/developer/app-store/apps/avatax/architecture.mdx:7]:
```suggestion
The AvaTax App uses the following webhook events for tax reporting and calculations:
```

File: versioned_docs/version-3.x/developer/app-store/apps/avatax/overview.mdx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+_AvaTax_ is a Saleor app that allows delegating tax calculations to [Avatax](https://www.avalara.com/us/en/products/calculations.html). It can replace the default ([flat rates](developer/taxes.mdx#flat-rates)) tax calculation method in Saleor. It affects both the checkout and the order creation process.
+
+Avalara AvaTax is a cloud-based solution automating transaction tax calculations and the tax filing process. Avalara
+provides real-time tax calculation using tax content from more than 12,000 US taxing jurisdictions and over 200
+countries, insuring your transaction tax is calculated based on the most current tax rules.
+
+## Features
+
+Taxes App offers:
```
Comment (peelar) [inline versioned_docs/version-3.x/developer/app-store/apps/avatax/overview.mdx:21]:
```suggestion
AvaTax App offers:
```

File: versioned_docs/version-3.x/developer/app-store/apps/avatax/overview.mdx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+If you want to explore how the AvaTax App works, you can read the [Avatax App architecture](./architecture) article.
+
+## Configuration
+
+### Channels
+
+For a channel to appear on the _Available channels_ list, you must first select **"use tax app"** as its tax calculation method. You can do that on the _Tax Configuration_ page (_Configuration_ → _Taxes_).
+
+In the _Taxes_ page, select the channel and change the value of _Select the method of tax calculation_ to "_Use tax app_". Then save it and come back to the Taxes App.
```
Comment (peelar) [inline versioned_docs/version-3.x/developer/app-store/apps/avatax/overview.mdx:44]:
```suggestion
In the _Taxes_ page, select the channel and change the value of _Select the method of tax calculation_ to "_Use tax app_". Then save it and come back to the AvaTax App.
```

**Review by peelar — APPROVED** [review]

**Pull request merged**